### DISCUSSION TITLE
To the Craft team: No feature request, but a compliment!

### DISCUSSION THREAD

**Discussion by bencresty** [craftcms/cms#10640] [General]
Walking through the incredible long list of changes, improvements and bugfixes of Craft now after not updating for a little while and it's quite impressive! You guys have been working hard, damn!
Thanks a lot for all the hard and great work!

**Comment by brandonkelly** [comment]
Aw thanks @bencresty!

**Comment by theskyfloor** [comment]
I will second this... its a slow burn... but instead of attrition the product is getting incrementally better every release.

**Comment by thupsi** [comment]
As a designer who also does front-end code (at least I try), and likes hacking and building things, I love Craft:
It is approachable for people like me that are not full-stack super devs, yet it feels flexible and powerfull enough for anything you can throw at it. I also really appreciate the blank-slate approach for the front-end, templating and content modeling, and the amount of customization you can do via the UI right on the CP, without messing with YAML files and stuff (personal preference, I know).

But maybe the biggest strength of Craft is the attitude of the people behind it and the surrounding community. It really feels like a safe and welcoming space. 

Thanks everyone and keep up the good work! ❤️
